.travis.yml

```diff
@@ -31,5 +35,5 @@ script:
   - make install
 
   # Test without any coverage
-  - make test-verbose
+  - make test-throughout
```

Comment:
Yes, and as discussed I will rename it to "test-full" since that makes more sense anyway.